### DISCUSSION TITLE
Update moor_flutter

### DISCUSSION THREAD

**Discussion by davidmartos96** [simolus3/drift#1119] [General]
@simolus3 Hey there! Any plans to update to update the moor_flutter package to stable null safety, or is anything else missing?

There is an extra required change, because Alex tweaked the API of sqflite recently. Now the `getDatabasesPath` output cannot be null, so `moor_flutter` is complaining with this:

```
../../../../.pub-cache/hosted/pub.dartlang.org/moor_flutter-4.0.0-nullsafety/lib/moor_flutter.dart:52:28: Warning: Operand of null-aware operation '!' has type 'String' which excludes null.
      resolvedPath = join((await s.getDatabasesPath())!, path); 
```

**Comment by simolus3** [comment]
Thanks for the pointers! I've just released `moor_flutter` version 4.0.0 which uses non-prerelease dependencies.
